src/elements/content-sidebar/additional-tabs/AdditionalTab.js

```diff
@@ -18,20 +18,51 @@ type Props = {
     onImageLoad: () => void,
 } & AdditionalSidebarTab;
 
-const AdditionalTab = ({ callback: callbackFn = noop, iconUrl, id, isLoading, onImageLoad, title, ...rest }: Props) => {
-    const className = classNames('bdl-AdditionalTab', { 'bdl-AdditionalTab--hidden': isLoading });
-
-    return (
-        <Tooltip position="middle-left" text={title}>
-            <PlainButton className={className} type="button" onClick={() => callbackFn({ id, callbackData: rest })}>
-                {id && id > 0 && iconUrl ? (
-                    <img className="bdl-AdditionalTab-icon" src={iconUrl} onLoad={onImageLoad} alt={title} />
-                ) : (
-                    <IconEllipsis className="bdl-AdditionalTab-moreIcon" />
-                )}
-            </PlainButton>
-        </Tooltip>
-    );
+type State = {
+    isErrored: boolean,
 };
 
+class AdditionalTab extends React.PureComponent<Props, State> {
+    state = {
+        isErrored: false,
+    };
+
+    onImageError = () => {
+        this.props.onImageLoad();
+        this.setState({ isErrored: true });
+    };
+
+    render() {
+        const { callback: callbackFn, id, isLoading, iconUrl, onImageLoad, title, ...rest } = this.props;
+        const { isErrored } = this.state;
+        const className = classNames('bdl-AdditionalTab', { 'bdl-AdditionalTab--hidden': isLoading });
+
+        let TabIcon;
+
+        if (isErrored) {
+            TabIcon = <AdditionalTabPlaceholder isLoading={false} />;
+        } else if (id && id > 0 && iconUrl) {
```

Comment:
Following this logic, if the user supplies `undefined` or `''` as an `iconUrl`, it'll display the  `IconEllipsis`.

Is this intended, or should we have a missing url icon/use the `AdditionalTabPlaceholder`?